src/main/java/inventorysetups/InventorySetupsPlugin.java

```diff
@@ -1036,12 +1038,36 @@ private boolean isNewPositionInvalid(int oldPosition, int newPosition, int size)
 
 	public List<InventorySetup> filterSetups(String textToFilter)
 	{
-		final String textToFilterLower = textToFilter.toLowerCase();
 		return inventorySetups.stream()
-			.filter(i -> i.getName().toLowerCase().contains(textToFilterLower))
+			.filter(inventorySetup -> setupContains(inventorySetup, textToFilter.toLowerCase()))
 			.collect(Collectors.toList());
 	}
 
+	private static boolean setupContains(InventorySetup inventorySetup, String textToFilterLower)
+	{
+		if (textToFilterLower.startsWith(ITEM_SEARCH_TAG) && textToFilterLower.length() > ITEM_SEARCH_TAG.length())
+		{
+			String itemName = textToFilterLower.trim().substring(ITEM_SEARCH_TAG.length());
```

Comment:
The `trim` before here is good but you also want to trim after. I think this still fails if someone does something like 

`   item: hammer  `. You are trimming to `item: hammer` but then fetch the string ` hammer` which has a space at the beginning.

Reply:
I didn't test this but this is what I think it does, lmk if my understanding is wrong.

Reply:
Good catch.